gatsby-browser.js

```diff
@@ -111,6 +165,10 @@ export const onClientEntry = () => {
             sampleRate: sample_rate,
         })
     }
+
+    if (isProduction() && !has_initialized) {
```

Comment:
What is the use of `has_initialized` yeah? As it just seems to be declared as false.